frontend/components/Filter/FilterBanner.tsx

```diff
@@ -207,6 +207,7 @@ export default function FilterBanner({
     closeModal();
   }
 
+  // Runs after first render
```

Comment:
what does this mean 😕 ❓

Reply:
idk i dont think i touched this - did you write it on my pc?